shared/src/main/java/com/faforever/neroxis/mask/FloatMask.java

```diff
@@ -228,7 +274,11 @@ Vector3 calculateNormalAt(int x, int y, float scale) {
      * @param scale Multiplicative factor for the noise
      */
     public FloatMask addGaussianNoise(float scale) {
-        return addPrimitiveWithSymmetry(SymmetryType.SPAWN, (x, y) -> (float) random.nextGaussian() * scale);
+        if (symmetrySettings.getSymmetry(SymmetryType.SPAWN).isOddSymmetry()) {
+            return apply((x, y) -> addPrimitiveAt(x, y, (float) random.nextGaussian() * scale));
+        } else {
+            return addPrimitiveWithSymmetry(SymmetryType.SPAWN, (x, y) -> (float) random.nextGaussian() * scale);
+        }
```

Comment:
This ends up breaking the symmetry guarantees because a new random is used for every x,y